src/views/v2/event-status/postponed-label.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Reason container for a postponed event.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/events-status/postponed-label.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://evnt.is/1aiy
+ *
+ * @version TBD
+ *
+ * @var \WP_Post      $event         The event post object with properties added by the `tribe_get_event` function.
+ * @var Status_Labels $status_labels An instance of the statuses handler.
+ *
+ * @see     tribe_get_event() For the format of the event object.
+ */
+namespace Tribe\Events\Event_Status;
+
+if ( 'postponed' !== $event->event_status ) {
+	return;
+}
+
+?>
+<span class="tribe-events-status__label tribe-events-status-__label-postponed">
+	<?php echo esc_html( $status_labels->get_postponed_label() ); ?>
+</span>
```

Comment:
If we follow the block from the `status-label` template, we'd have something like the following:

```suggestion
<span class="tribe-events-status-label__text tribe-events-status-label__text--postponed">
	<?php echo esc_html( $status_labels->get_postponed_label() ); ?>
</span>
```